src/angular-app/languageforge/lexicon/shared/share-with-others/role-dropdown.component.html

```diff
@@ -8,9 +8,15 @@
       <i class="fa fa-{{ roleDetail.icon }}"></i>
       {{ roleDetail.description }} <i class="fa fa-check" ng-show="$ctrl.selectedRoleDetail == roleDetail"></i>
     </a>
-    <div class="dropdown-divider" ng-if="$ctrl.allowDelete"></div>
-    <li class="dropdown-item" role="menuitem" ng-if="$ctrl.allowDelete">
-      <div ng-click="$ctrl.onDeleteTarget({$event: {target: $ctrl.target}})" class="text-danger">remove</div>
-    </li>
+    <div class="dropdown-divider" ng-if="$ctrl.currentUserIsOwnerOrAdmin() && !userIsInvitee({$event: {target: $ctrl.target}})"></div>
```

Comment:
> `ng-if="$ctrl.currentUserIsOwnerOrAdmin() && !userIsInvitee(`
To me this line tells me that the "make owner" option is not visible if the user is only "invited".

I'm confused then how one would get to the "chicken page" where it says invitees are not allowed to be owners (yet).

Reply:
this was removed in the latest commit